src/l500/l500-color.cpp

```diff
@@ -358,6 +368,33 @@ namespace librealsense
         if (_state != sensor_state::OWNED_BY_USER)
             throw wrong_api_call_sequence_exception("tried to start an unopened sensor");
 
+#ifdef __ANDROID__
+        if (supports_option(RS2_OPTION_ENABLE_WEAK_USB_HOST_WA))
+        {
+            auto usb_perf_enabled = get_option(RS2_OPTION_ENABLE_WEAK_USB_HOST_WA).query();
+
+            if (usb_perf_enabled)
+            {
+                // TPROC USB Granularity and TRB threshold settings for improved performance and stability
```

Comment:
Please include the ticket number (RS5-8011), both here and in the depth sensor, so we can go back and read what this is all about

Reply:
included ticket number